src/include/btmem.h

```diff
@@ -1601,16 +1605,18 @@ struct __wt_update {
 
 /* When introducing a new flag, consider adding it to WT_UPDATE_SELECT_FOR_DS. */
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_UPDATE_DS 0x01u                       /* Update has been written to the data store. */
-#define WT_UPDATE_HS 0x02u                       /* Update has been written to history store. */
-#define WT_UPDATE_PREPARE_RESTORED_FROM_DS 0x04u /* Prepared update restored from data store. */
-#define WT_UPDATE_RESTORED_FAST_TRUNCATE 0x08u   /* Fast truncate instantiation */
-#define WT_UPDATE_RESTORED_FROM_DS 0x10u         /* Update restored from data store. */
-#define WT_UPDATE_RESTORED_FROM_HS 0x20u         /* Update restored from history store. */
-#define WT_UPDATE_RTS_DRYRUN_ABORT 0x40u         /* Used by dry run to mark a would-be abort. */
-#define WT_UPDATE_TO_DELETE_FROM_HS 0x80u        /* Update needs to be deleted from history store */
-                                                 /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
-    uint8_t flags;
+#define WT_UPDATE_DS 0x001u                       /* Update has been chosen to the data store. */
+#define WT_UPDATE_DURABLE 0x002u                  /* Update has been durable. */
+#define WT_UPDATE_HS 0x004u                       /* Update has been written to history store. */
+#define WT_UPDATE_PREPARE_RESTORED_FROM_DS 0x008u /* Prepared update restored from data store. */
+#define WT_UPDATE_RESTORED_FAST_TRUNCATE 0x010u   /* Fast truncate instantiation */
+#define WT_UPDATE_RESTORED_FROM_DELTA 0x020u      /* Update restored from delta. */
+#define WT_UPDATE_RESTORED_FROM_DS 0x040u         /* Update restored from data store. */
+#define WT_UPDATE_RESTORED_FROM_HS 0x080u         /* Update restored from history store. */
+#define WT_UPDATE_RTS_DRYRUN_ABORT 0x100u         /* Used by dry run to mark a would-be abort. */
+#define WT_UPDATE_TO_DELETE_FROM_HS 0x200u /* Update needs to be deleted from history store */
+                                           /* AUTOMATIC FLAG VALUE GENERATION STOP 16 */
```

Comment:
This is a future thing. We can optimize to reduce the size of the flags. For WT_UPDATE_RESTORED_FROM_DELTA, we don't need it once we can generate a full disk image directly from deltas. We need to get rid of another flag to reduce the size here.